common/src/java/com/zimbra/common/util/DateParser.java

```diff
@@ -42,13 +44,22 @@ public Date parse(String s) {
     public String format(Date date) {
         return getFormatter().format(date);
     }
-    
```

Comment:
nit: please do not remove a blank line for readability.

Reply:
Done